server.js

```diff
@@ -0,0 +1,119 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const pug = require('pug');
+const log = require('./log')(module);
+
+const app = express();
+const router = express.Router();
+
+app.set('view engine', 'pug');
+
+router.use(function (req, res, next) {
+  log.info(req.method, `${req.protocol}://${req.get('host')}${req.originalUrl}`);
+  next();
+});
+
+app.use(router);
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+let blogs = [
```

Comment:
export this into mock folder to blogs.mock.js, please

Reply:
Fixed